TaskEngine/Tasks/UpdateBoxTokenTask.cs

```diff
@@ -17,9 +17,10 @@ public UpdateBoxTokenTask(RabbitMQConnection rabbitMQ, BoxAPI box, ILogger<Updat
             _box = box;
         }
 #pragma warning disable 1998
-        protected async override Task OnConsume(string emptyString, TaskParameters taskParameters)
+        protected async override Task OnConsume(string emptyString, TaskParameters taskParameters, ClientActiveTasks cleanup)
         {
+            //registerTask(cleanup, "RefreshAccessTokenAsync"); // may throw AlreadyInProgress exception
```

Comment:
Is this supposed to be commented? Just want to check because all of the other similar tasks do not have this commented.

Reply:
Yes. Added more comments and made it definitely commented.
            // Maybe in the future if we use this task again: registerTask(cleanup, "RefreshAccessTokenAsync"); // may throw AlreadyInProgress exception
            // no. xx nope await _box.RefreshAccessTokenAsync();
            // refreshing the Box access token caused the token to go stale
            // We've had a better experience not refreshing it

Reply:
Yes it is supposed to be commented. Addressed in https://github.com/classtranscribe/WebAPI/pull/101/commits/76621f50fd54454f36a5e3d6121bc7d156775524